app/services/inherited_proofing/va/service.rb

```diff
@@ -0,0 +1,86 @@
+module InheritedProofing
+  module Va
+    # Encapsulates request, response, error handling, validation, etc. for calling
+    # the VA service to gain PII for a particular user that will be subsequently
+    # used to proof the user using inherited proofing.
+    class Service
+      BASE_URI = Rails.configuration.inherited_proofing[:va][:base_uri].freeze
```

Comment:
AFAIK we don't use Rails.configuration like this, we tend to use our own [IdentityConfig](https://handbook.login.gov/articles/appdev-secrets-configuration.html#configuration-in-rails-apps) so we can integrate them better

Reply:
done